<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataInputClass</class>
 <widget class="QDialog" name="DataInputClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>345</width>
    <height>268</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>345</width>
    <height>268</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>345</width>
    <height>268</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>DataInput</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>190</y>
     <width>193</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QPushButton" name="cancelBtn">
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="sureBtn">
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>100</y>
     <width>301</width>
     <height>64</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>缓冲区半径: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="radiusEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>输出文件:   </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="fileEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="fileBtn">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="TigerGIS.qrc">
          <normaloff>:/TigerGIS/res/folderOpen.png</normaloff>:/TigerGIS/res/folderOpen.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="openLabel2">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>65</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>输入图层： </string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>118</x>
     <y>31</y>
     <width>201</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QComboBox" name="overlayCambo"/>
    </item>
    <item>
     <widget class="QPushButton" name="fileButton1">
      <property name="minimumSize">
       <size>
        <width>21</width>
        <height>23</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>21</width>
        <height>23</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="TigerGIS.qrc">
        <normaloff>:/TigerGIS/res/folderOpen.png</normaloff>:/TigerGIS/res/folderOpen.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="openLabel1">
   <property name="geometry">
    <rect>
     <x>23</x>
     <y>31</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>叠加图层： </string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>60</y>
     <width>201</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QComboBox" name="inputCombo"/>
    </item>
    <item>
     <widget class="QPushButton" name="fileButton2">
      <property name="minimumSize">
       <size>
        <width>21</width>
        <height>23</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>21</width>
        <height>23</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="TigerGIS.qrc">
        <normaloff>:/TigerGIS/res/folderOpen.png</normaloff>:/TigerGIS/res/folderOpen.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="TigerGIS.qrc"/>
 </resources>
 <connections/>
</ui>
